app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,25 @@
  *= require_tree .
  *= require_self
  */
+
```

Comment:
Better to create a new css file to sit in the folder rather than edit the manifest to add styles.